### ISSUE TITLE
Create +use bad exposure list for Bok

### ISSUE THREAD

**Issue by djschlegel** [legacysurvey/legacyzpts#55]
We already have a bad exposure list for decam and mosaic, and should make one for bok as well and put in the same location:
  https://github.com/legacysurvey/legacyzpts/tree/master/py/legacyzpts/data

Jinyi Yang has provided a bad exposure list on March 21, 2019, with this note: "These tiles are not meet the depth requirements or do not have catalogs. The list is based on the summary of tiles observed before 2019 given by BASS team."   This file attached below, although I don't think we should start with this since it doesn't add any information other than listing  686 exposures that are shallow.  Instead, let's look for problematic data such as the night of donuts listed in ticket #53 .

[90prime-bad_expid.txt](https://github.com/legacysurvey/legacyzpts/files/3008683/90prime-bad_expid.txt)

**Comment by moustakas** [comment]
@dstndstn and I have moved the legacy_zeropoints functionality to `legacypipe` (in the `import-legacyzpts` branch), so new issues pertaining to zeropoints should be raised there.  

The initial 90prime bad exposure list was added in https://github.com/legacysurvey/legacypipe/commit/0364eb6147b9074970218750140e3c28ace25ebe with just the night of out-of-focus data identified in #53.

Do we need to do anything else with the ~686 exposures in the file linked here?
